kmip/core/attributes.py

```diff
@@ -154,20 +153,17 @@ def create(cls, name_value, name_type):
 # 3.3
```

Comment:
nit:
Instead of having only the chapter number for the kmip spec, we should specifically state "KMIP Spec 1.1". Even more so that there are different versions and we don't know how this might change in the future. I would suggest we say "KMIP Spec 1.1: 3.3"